packages/fxa-settings/src/components/GeneralizedRootApp/en-US.ftl

```diff
@@ -0,0 +1,45 @@
+## Firefox and Mozilla Brand
```

Comment:
I think you're going to want to modify the way our Fluent stuff gets concatenated with this change (it shouldn't be hard).

Right now, our branding file is only rendered at the top of the Fluent file because it's in `App/index.tsx` and it's the first component alphabetically. See [this section of this doc ](https://docs.google.com/document/d/1TgD7WBLvS4aZ10neLJNk-EYvDR8Kb9GOk2KHxhjVzbk/edit#bookmark=id.jjr6o96gv1h1) and related issues [here](https://mozilla-hub.atlassian.net/browse/FXA-5994) and [here](https://mozilla-hub.atlassian.net/browse/FXA-5995). You might want to just temporarily add the branding stuff to `.license.header`, check out our `.rescriptsrc.js` file for `mergeFtlPlugin`. I think if you run a build you can see the output.